docs/zh/UserGuide/Maintenance-Tools/Sync-Tool.md

```diff
@@ -31,7 +31,7 @@
 
 ## 2.模型定义
 
-![img](https://y8dp9fjm8f.feishu.cn/space/api/box/stream/download/asynccode/?code=ODYwOTUxMGI4YjI0N2FlOWFlZmI4MDcxNDlmZDE1MGZfQ1hzcVM1bXJxUkthd1hta3hSaDdnZW1aZHh2cE5iTUxfVG9rZW46Ym94Y25CRjF5dXd0QkVEYzVrSnJHcUdkd3VlXzE2NDkwNTkxNDc6MTY0OTA2Mjc0N19WNA)
+![img](https://y8dp9fjm8f.feishu.cn/space/api/box/stream/download/asynccode/?code=ZjhhODI1MjkyMjQyMmMyNjdmYTIxOTE0MmU5M2U2MzRfNHAzS055MVZwWVltNU5WcnJXN2FxbHZoU0s4UHhxWTdfVG9rZW46Ym94Y25CRjF5dXd0QkVEYzVrSnJHcUdkd3VlXzE2NTA5NzA2ODI6MTY1MDk3NDI4Ml9WNA)
```

Comment:
Using feishu picture is not a good choice.
You can upload the picture to https://github.com/apache/iotdb-bin-resources instead.

Reply:
done